### PR TITLE
Adding a file for Dublin Ireland

### LINKED ISSUE #12: Hometown for kenmoran1980
# Add file for Dublin, Ireland

1. Do GitHub stuff

- [ ] 2

- [ ] 3

### PR DESCRIPTION
**Pull request by kenmoran1980**
This PR closes #12 by fixing #12 

@acorkery or @KyleWatkinson broke it :-1:

### CODE REVIEW

File: places/Dublin-Ireland.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Dublin
+
+###Places to drink
+-LOADS!!!
+-Stay away from Temple Bar
```
Comment (KyleWatkinson) [inline places/Dublin-Ireland.md]: I disagree with this.

Comment (acorkery) [inline places/Dublin-Ireland.md]: (if you know what's good for you)

Comment (kenmoran1980) [inline places/Dublin-Ireland.md]: Who cares

Comment (douglasreis) [inline places/Dublin-Ireland.md]: Anything else to do that is not drinking related? :smile:

Comment (kenmoran1980) [inline places/Dublin-Ireland.md]: There's a few but not worth worrying about. It's always raining :)

**Pull request merged**